frontend/src/queries/schema.ts

```diff
@@ -668,6 +668,7 @@ export interface ChartSettingsFormatting {
 }
 
 export interface ChartSettingsDisplay {
+    color?: string
```

Comment:
There should be a backend equivalent for this - generate it with `pnpm schema:build` command. Without it, I think saving an insight with a color will get rejected by the backend (I think)

Reply:
Ah right, I hadn't tested saving an insight. Will try what you said 👍

Reply:
that should be fixed now